test/es6/module-syntax.js

```diff
@@ -0,0 +1,108 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// ES6 Module syntax tests -- verifies syntax of import and export statements
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+    {
+        name: "All valid (non-default) export statements",
+        body: function () {
+            assert.doesNotThrow(function () { WScript.LoadModuleFile('.\\module\\ValidExportStatements.js', 'samethread'); }, "Valid export statements");
+        }
+    },
+    {
+        name: "Valid default export statements",
+        body: function () {
+            assert.doesNotThrow(function () { WScript.LoadModuleFile('.\\module\\ValidExportDefaultStatement1.js', 'samethread'); }, "Unnamed function expression default export");
```

Comment:
For test readability would it make sense to have a LoadModuleScript api that takes a string of the module contents rather than loading a file?

With the tests in separate individual files its hard to look at them all together to compare and contrast.